src/extensions/codemirror/CodeMirror.tsx

```diff
@@ -98,108 +83,95 @@ export interface CodeEditorProps {
     /**
      *  handler for scroll event
      */
-    onScroll?: (editorInstance: CodeMirror.Editor) => void;
+    onScroll?: (event: Event, view: EditorView) => boolean | void;
     /**
      * optional property to fold code for the supported modes e.g: xml, json etc.
      */
     supportCodeFolding?: boolean;
+    /**
+     * highlight active line where the cursor is currently in
+     */
+    shouldHighlightActiveLine?: boolean;
 }
 
-/**
```

Comment:
Why was the documentation removed here? This is displayed automatically in our storybook. Feel free to improve and extend it but please do not remove it.